string hand_name
int32 action_type
--------------------------------
bool success
